src/lib_c/x86_64-windows-msvc/c/iocp.cr

```diff
@@ -0,0 +1,53 @@
+require "c/winnt"
+
+# WSAOVERLAPPED is the primary communication structure for async I/O on Windows.
+# See https://docs.microsoft.com/en-us/windows/win32/api/winsock2/ns-winsock2-wsaoverlapped
+@[Extern]
+struct WSAOVERLAPPED
+  internal : LibC::ULONG_PTR
+  internalHigh : LibC::ULONG_PTR
+  offset : LibC::DWORD
+  offsetHigh : LibC::DWORD
+  hEvent : LibC::HANDLE
+  property cEvent : Void*
+
+  def initialize(crystal_event : Crystal::Event)
+    @cEvent = crystal_event.unsafe_as(Pointer(Void))
+  end
+end
+
+@[Link("advapi32")]
+lib LibC
```

Comment:
These are not part of LibC. Should there be a new lib for these? `LibAdv` perhaps.

Reply:
No no, this is the convention thus far. Everything in winapi also goes to `LibC`.
https://github.com/crystal-lang/crystal/search?q=advapi32